apps/src/craft/designer/craft.js

```diff
@@ -0,0 +1,913 @@
+    winAvatar: MEDIA_URL + "Events/Pop_Up_Character_Chicken_Win.png",
+  }
+};
+
+var interfaceImages = {
+  DEFAULT: [
+    MEDIA_URL + "Sliced_Parts/MC_Loading_Spinner.gif",
+    MEDIA_URL + "Sliced_Parts/Frame_Large_Plus_Logo.png",
+    MEDIA_URL + "Sliced_Parts/Frame_Large_Plus_LogoNew.png",
+    MEDIA_URL + "Sliced_Parts/Pop_Up_Slice.png",
+    MEDIA_URL + "Sliced_Parts/X_Button.png",
+    MEDIA_URL + "Sliced_Parts/Button_Grey_Slice.png",
+    MEDIA_URL + "Sliced_Parts/Run_Button_Up_Slice.png",
+    MEDIA_URL + "Sliced_Parts/MC_Run_Arrow_Icon.png",
+    MEDIA_URL + "Sliced_Parts/Run_Button_Down_Slice.png",
+    MEDIA_URL + "Sliced_Parts/Reset_Button_Up_Slice.png",
+    MEDIA_URL + "Sliced_Parts/MC_Reset_Arrow_Icon.png",
+    MEDIA_URL + "Sliced_Parts/Reset_Button_Down_Slice.png",
+    MEDIA_URL + "Sliced_Parts/Callout_Tail.png",
+  ],
+  1: [
+    MEDIA_URL + "Sliced_Parts/Steve_Character_Select.png",
+    MEDIA_URL + "Sliced_Parts/Alex_Character_Select.png",
+    characters.Steve.staticAvatar,
+    characters.Steve.smallStaticAvatar,
+    characters.Alex.staticAvatar,
+    characters.Alex.smallStaticAvatar,
+  ],
+  2: [
+    // TODO(bjordan): find different pre-load point for feedback images,
+    // bucket by selected character
+    characters.Alex.winAvatar,
+    characters.Steve.winAvatar,
+    characters.Alex.failureAvatar,
+    characters.Steve.failureAvatar,
+  ],
+  6: [
+  ]
+};
+
+var MUSIC_METADATA = [
+  {volume: 1, hasOgg: true, name: "vignette1", group: 'day'},
+  {volume: 1, hasOgg: true, name: "vignette2-quiet", group: 'night'},
+  {volume: 1, hasOgg: true, name: "vignette3", group: 'night'},
+  {volume: 1, hasOgg: true, name: "vignette4-intro", group: 'day'},
+  {volume: 1, hasOgg: true, name: "vignette5-shortpiano", group: 'day'},
+  {volume: 1, hasOgg: true, name: "vignette7-funky-chirps-short", group: 'day'},
+  {volume: 1, hasOgg: true, name: "vignette8-free-play", group: 'day'},
+];
+
+var CHARACTER_STEVE = 'Steve';
+var CHARACTER_ALEX = 'Alex';
+var DEFAULT_CHARACTER = CHARACTER_STEVE;
+
+function trySetLocalStorageItem(key, value) {
+  try {
+    window.localStorage.setItem(key, value);
+  } catch (e) {
+    /**
+     * localstorage .setItem in iOS Safari Private Mode always causes an
+     * exception, see http://stackoverflow.com/a/14555361
+     */
+    if (console && console.log) {
+      console.log("Couldn't set local storage item for key " + key);
+    }
+  }
+}
+
+/**
+ * Initialize Blockly and the Craft app. Called on page load.
+ */
+Craft.init = function (config) {
+
+  if (config.level.puzzle_number === 1 && config.level.stage_total === 1) {
+    // Not viewing level within a script, bump puzzle # to unused one so
+    // asset loading system and levelbuilder overrides don't think this is
+    // level 1 or any other special level.
+    config.level.puzzle_number = 999;
+  }
+
+  if (config.level.isTestLevel) {
+    config.level.customSlowMotion = 0.1;
+  }
```

Comment:
prob doesnt matter enough to bother changing, but i wonder if a better approach for this would be to use query params